waymo_raw_data_to_argoverse.py

```diff
@@ -84,17 +101,28 @@ def save_json_dict(json_fpath: Union[str, "os.PathLike[str]"], dictionary: Dict[
 	with open(json_fpath, "w") as f:
 		json.dump(dictionary, f)
 
+def get_log_id_from_files(files, record_dir):
+	log_ids = []
+	for file in files:
+		file = file.strip(record_dir)
+		file = file.strip("segment-")
+		file = file.strip("with_camera_labels.tfrecord")
+		log_ids.append(file)
+	return log_ids
 
-def main(save_images: bool, save_poses: bool, save_calibration: bool):
+def main(save_images: bool, save_poses: bool, save_calibration: bool, save_cloud: bool, save_labels:bool):
```

Comment:
i recommend we keep line length down to 80 chars so that it's a bit more readable:
https://google.github.io/styleguide/pyguide.html#32-line-length

Do you mind adding type hints as well? e.g. per https://google.github.io/styleguide/pyguide.html#221-type-annotated-code

Static analysis of types is possible with http://mypy-lang.org/
https://github.com/python/mypy

Reply:
Addressed in [this commit](https://github.com/johnwlambert/waymo_to_argoverse/pull/2/commits/8931614c713c462a1efbbc0f06a366d44a28df25)